sdcontent/INSTRUCTIONS-README.txt

```diff
@@ -1,34 +1,38 @@
       iii. Select "Overwrite Format"
-      iv. Check that the SD card you inserted matches the one selected by the Tool
+      iv. Check that the SD card you inserted matches the one selected by the tool
       v. Click the "Format" button
    c. Linux
-      i. We recommend using gparted (or the command line version parted)
-      ii. Format the entire disk as FAT32
+      i. We recommend using gparted (or the command line version, parted)
+      ii. Format the entire card as FAT32
    d. Cards over 32GB
-      i. Follow the instructions on https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
+      i. Follow the instructions at https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
 3. Extract the files contained in this NOOBS zip file.
-4. Copy the extracted files onto the SD card that you just formatted so that this file is at the root directory of the SD card. Please note that in some cases it may extract the files into a folder, if this is the case then please copy across the files from inside the folder rather than the folder itself.
-5. Insert the SD card into your Pi and connect the power supply.
+4. Copy the extracted files onto the SD card that you just formatted. The files must be in the root directory of the SD card, not in a subdirectory within this.
+5. Insert the SD card into the Raspberry Pi and connect the power supply.
+
+**Raspberry Pi 4B**:
+- If you are using an HDMI display,  you must connect it to the micro HDMI port closest to the power jack, labelled HDMI 0.
+- If you are instead connecting a composite video display, you must first edit the file config.txt
+- You cannot use both an HDMI and composite display at the same time
 
-**Raspberry Pi 4 Model B:** display must be connected to HDMI port closest to power jack.
 
 Your Pi will now boot into NOOBS and should display a list of operating systems that you can choose to install.
 If your display remains blank, you should select the correct output mode for your display by pressing one of the following number keys on your keyboard:
 1. HDMI mode - this is the default display mode.
 2. HDMI safe mode - select this mode if you are using the HDMI connector and cannot see anything on screen when the Pi has booted.
-3. Composite PAL mode - select either this mode or composite NTSC mode if you are using the composite RCA video connector.
+3. Composite PAL mode - select either this mode or composite NTSC mode if you are using a screen connected to the composite video output. On most Raspberry Pis the composite video output is on the 4-pole 3.5mm jack, along with stereo audio. On the original Raspberry Pi, released in 2012, composite video is output on the RCA connector. Not all 4-pole 3.5mm cables are suitable for using to connect composite video; see the [Monitor Connection](https://www.raspberrypi.org/documentation/setup/monitor-connection.md) page for details of which cable to use.

```

Comment:
Probably doesn't make sense to use markdown link-syntax inside a plain-text file? :wink: